app/views/products/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<body>
+<main>
+<section>
+  <h2><%= @product.name %></h2>
+    <p>Description: <%= @product.description.capitalize %></p>
+    <p>Price: $<%= @product.price %></p>
+    <p>Stock: <%= @product.stock %> units</p>
+    <p>Available: <%= @product.available %></p>
+    <p>User_id: <%= @product.user_id %></p>
+    <img src= <%= @product.photo_url %>>
+    
```

Comment:
Yay for sections! I'm wondering if the first section should end here since all of the content is already wrapped between 'main'.